kolena/_experimental/object_detection/workflow.py

```diff
@@ -88,6 +90,7 @@ class TestCaseMetricsSingleClass(MetricsTestCase):
     TP: int
     FN: int
     FP: int
+    nIgnored: int
```

Comment:
Flagging that this is a bit out of place as the only camelCased field here, but we do have `nImages` in the  nested test case metrics, so this would be consistent with that.